instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/HttpClientTracer.java

```diff
@@ -32,6 +32,18 @@
 
   protected static final String USER_AGENT = "User-Agent";
 
+  protected HttpClientTracer() {
+    super();
+  }
+
+  protected HttpClientTracer(Tracer tracer) {
```

Comment:
```suggestion
  @Deprecated
  protected HttpClientTracer(Tracer tracer) {
```

Reply:
yeah, I was trying to decide on whether to deprecate or not. If you think so, then I'm 💯  on that.

Reply:
done